.gitlab-ci.yml

```diff
@@ -48,19 +48,28 @@ lint_test_scripts:
 .build_environment_common: &build_environment_common
   <<: *parameters_common
   image: registry.hub.docker.com/library/docker:stable
+  before_script:
+    - |
+      apk update && apk add git
```

Comment:
git is not installed by default and we need it to check whether there are changes.

Reply:
When we rebuild Dave, we can make sure these are present in the runner.